packages/main/src/WheelSlider.hbs

```diff
@@ -20,11 +21,11 @@
 	<div id="{{this._id}}--inner" class="ui5-wheelslider-inner">
 	<div id="{{this._id}}--selection-frame" class="ui5-wheelslider-selection-frame">
 		</div>
-		<div id="{{this._id}}--wrapper" class="ui5-wheelslider-wrapper" @wheel="{{_handleWheel}}">
+		<div id="{{this._id}}--wrapper" class="ui5-wheelslider-wrapper">
 			{{#if _expanded}}
 				<ul id="{{this._id}}--items-list">
-					{{#each items}}
-						<li class="ui5-wheelslider-item">{{this}}</li>
+					{{#each showItems}}
```

Comment:
this sounds like a boolean. Should be "itemsToShow", "displayedItems", "allItems" etc...

Reply:
Done